### PR TITLE
Disable update-staging-client-go until code freeze

### PR DESCRIPTION
**Pull request by caesarxuchao**
I don't want to cause developer friction until code freeze. I'll babysit the script for the moment to keep the client-go up-to-date.

Should had done this in #34489.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35915)

### CODE REVIEW

File: hack/update-all.sh [hunk 1]
```diff
@@ -58,8 +58,8 @@ BASH_TARGETS="
 	generated-swagger-docs
 	swagger-spec
 	openapi-spec
-	api-reference-docs
-	staging-client-go"
+	api-reference-docs"
+#	staging-client-go"
```
Comment (mml) [inline hack/update-all.sh]: Add a comment like `TODO(caesarxuchao): uncomment after 1.5 code freeze`

**Review by mml — CHANGES_REQUESTED** [review]
LG otherwise.

**Comment by caesarxuchao** [comment]
Thanks. Done. Self applying the lgtm.

**Comment by caesarxuchao** [comment]
@k8s-bot node e2e test this

**Comment by caesarxuchao** [comment]
@k8s-bot node e2e test this.

**Pull request merged**